### ISSUE TITLE
Update UI component for Knock In App Notifications Display

### ISSUE THREAD

**Issue by dillchen** [hicommonwealth/commonwealth#9762]
**Description**

We currently use the default Knock Provider In App. We should create a custom component to update the notifications In App Display.

**Project Owner**  
@dillchen 

**User Stories / Acceptance Criteria**  
- Notifications Display, Link Handling, Clear All etc all work as is

**Design Devlink**  
https://www.figma.com/design/WdlVQRpQKg4GMgCOzbQt34/Notifications?node-id=604-224&t=alPU6pUtBFdRfroe-1

**Design Screenshot**  

**Old**

<img width="348" alt="image" src="https://github.com/user-attachments/assets/36380b67-0c14-49ae-8f39-37d6b88384f0">

<img width="400" alt="image" src="https://github.com/user-attachments/assets/de2dd4c9-e0e3-4347-ad39-666d8a9e9614">

**New**

![image](https://github.com/user-attachments/assets/38f14c6d-bb47-4b19-b130-0465e3a44c3c)

[To Add Link Here](https://www.figma.com/design/WdlVQRpQKg4GMgCOzbQt34/Notifications?node-id=604-224&t=P4uAyP2qxeEds1dT-4)

**Additional Context**  

You will have to build this ontop of the Knock SDK, please follow their documentation [link](https://docs.knock.app/in-app-ui/react/overview)

**Comment by KaleemNeslit** [comment]
@Israellund  is design confirm for this ticket ?

**Comment by Israellund** [comment]
@KaleemNeslit Not yet, a Figma link would be attached if so. I'll follow up with Design and see if they have anything for this.

**Comment by KaleemNeslit** [comment]
@Israellund  how do i enable notification , how do i get knockJWT and other related thing to start this ticket

**Comment by Israellund** [comment]
@KaleemNeslit I'm looking into any Knock env vars,  are you currently unable to start working on this?

**Comment by KaleemNeslit** [comment]
> @KaleemNeslit I'm looking into any Knock env vars, are you currently unable to start working on this?

yes i am not able start  work on this as i am not able to receive notification @Israellund

**Comment by Israellund** [comment]
@KaleemNeslit just sent you a message on slack with env vars, let me know if those work.

**Comment by KaleemNeslit** [comment]
@Israellund still not able to receive notification and not able to see notification icon

**Comment by Israellund** [comment]
> @Israellund still not able to receive notification and not able to see notification icon

Just sent you Knock env vars. Let me know if you still can't receive anything

**Comment by dillchen** [comment]
@KaleemNeslit, sorry I have to reopen the ticket, I linked to the wrong frame, please see updated screenshot
